client/src/pages/addtree/AddTree.js

```diff
@@ -13,16 +13,39 @@ import AddTreeModal from './AddTreeModal';
 let renderCount = 0;
 const currentMarkers = [];
 
-function AddTree({ map, setZoom, center, setCenter, setNewTreeAdded, newTreeAdded }) {
+function AddTree({
+  map, setZoom, center, setCenter, setNewTreeAdded, newTreeAdded, geolocater,
+}) {
   const { user, isAuthenticated, loginWithRedirect } = useAuth0();
   const [addTreeSelected, setAddTreeSelected] = useState(false);
   const mutateUser = useUserMutation();
 
+  const [showAddTreeModal, setShowAddTreeModal] = useState(false);
+
+  const handleGeolocation = () => {
+    geolocater.on('geolocate', (position) => {
+      const lat = position.coords.latitude;
+      const lng = position.coords.longitude;
+      console.log('lat, lng', lat, lng);
+      // setCenter([lng, lat]);
```

Comment:
Doesn't tapping the location button automatically fly the map the user's location?